<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find graphslam)/view.vcg" /> 
  
  <include file="$(find turtlebot_slam)/move_base_config/move_base.xml" />

  <node pkg="graphslam" type="sm_node" name="sm_node" output="screen">
  	<param name="scan_topic" value="base_scan" />
  	<param name="max_laser_range" value="4.6" />
  	<param name="min_laser_range" value="0.15" />
  </node>
  
  <node pkg="graphslam" type="graphslam" name="graphslam" output="screen">
  	<param name="resolution" value=".05" />
  	<param name="scan_topic" value="base_scan" />
  	<param name="max_laser_range" value="4.6" />
  	<param name="min_laser_range" value="0.15" />
  </node>

  <node pkg="turtlebot_slam" type="turtlebot_exploration" name="turtlebot_slam" output="screen" />
  <node pkg="slam_navigation" type="slam_navigation" name="slam_navigation" output="screen" />

</launch>